src/Reviews/ReviewAssignmentModel.cs

```diff
@@ -47,5 +50,21 @@ public class ReviewAssignmentModel
         /// Gets or sets the percentage of components reviewed by the reviewer.
         /// </summary>
         public virtual int PercentageCompleted { get; set; }
+
+        /// <summary>
+        /// Gets or sets the review state by name.
+        /// </summary>
+        public string StateName
+        {
+            get => this.State.ToString();
+
+            set => this.State = value.ToEnum<ReviewStateType>();
+        }
```

Comment:
I dont think we need this. State serializes fine to string and if not, the enum should have the Json string serializer attribute added. This seems redundant. If the UI uses this property, let's remove it, and instead use State.